hc/api/tests/test_ping.py

```diff
@@ -62,12 +64,31 @@ def test_it_reads_forwarded_protocol(self):
         r = self.client.get("/ping/%s/" % self.check.code,
                             HTTP_X_FORWARDED_PROTO="https")
         ping = Ping.objects.latest("id")
-        ### Assert the expected response status code and ping's scheme
+        # Assert the expected response status code and ping's scheme
+        self.assertEqual(r.status_code, 200)
+        self.assertEqual(ping.scheme, "https")
 
     def test_it_never_caches(self):
         r = self.client.get("/ping/%s/" % self.check.code)
         assert "no-cache" in r.get("Cache-Control")
 
-    ### Test that when a ping is made a check with a paused status changes status
-    ### Test that a post to a ping works
-    ### Test that the csrf_client head works
+    # Test that when a ping is made a check with a paused status changes status
+    # Test that a post to a ping works
+    # Test that the csrf_client head works
+
+    def test_it_changes_status_of_paused_check(self):
+        # paused check
+        self.check.status = "paused"
+        self.check.save()
+        r = self.client.get("/ping/%s/" % self.check.code)
+        self.check.refresh_from_db()
+        ping = Ping.objects.latest("id")
+        self.assertEqual(self.check.status, "up")
+
+    def test_it_works_with_post_request(self):
+        r = self.client.post("/ping/%s/" % self.check.code)
+        self.assertEqual(r.status_code, 200)
+
+    def test_it_works_with_csrf_client_head(self):
+        csrf_client = Client(enforce_csrf_checks=True)
+        assert csrf_client
```

Comment:
The meaning of this line isn't clear to me. We should use instance methods for assertions in the Django TestCase class.